src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue2222.cs

```diff
@@ -1,5 +1,4 @@
-﻿#if TEST_FAILS_ON_IOS //Application crashes when ToolbarItem is created with invalid IconImageSource name.Issue Link: https://github.com/dotnet/maui/issues/27095
-using NUnit.Framework;
+﻿using NUnit.Framework;
```

Comment:
What happens on other platforms? Based on the test, guess not throwing an exception, right?